src/filler.ts

```diff
@@ -159,11 +160,11 @@ const commanderParseInt = (value: string, _: any) => {
 }
 
 (async () => {
-    const config = require(`./config`);
+    //const config = require(`./config`);
```

Comment:
we could remove this line if it's not used anymore

Reply:
done